actionview/CHANGELOG.md

```diff
@@ -106,6 +106,11 @@
           <%= f.label :title %>
           <%= f.text_field :title, aria: { describedby: f.field_id(:title, :error) } %>
           <%= tag.span("is blank", id: f.field_id(:title, :error) %>
+
+          <%= f.fields :tag_attributes, model: @post.tags.new do |tf| %>
+            <%= tf.label :name %>
+            <%= tf.text_field :name, aria: { describedby: tf.field_id(:name, :error) } %>
+          <% end %>
```

Comment:
This example is getting pretty big.
I'm not sure it's getting clearer with the added nested fields...

Reply:
That's a good point. Since this change fixes a bug introduced in the original, and the original is not yet released, it might not be worth mentioning at all.